.travis.yml

```diff
@@ -90,18 +90,26 @@ jobs:
       if: branch in (master, develop)
       python: 3.6
       install:
-        - pip install -r requirements/deploy.txt
-        - pip install --editable . # install wam, as needed by GB
+        - sudo docker pull ${DOCKER_USER}/${DOCKER_REPO}
       script:
-        # create standalone application
-        - gravitybee --src-dir src --sha file --with-latest --extra-data static --verbose
+        - |
```

Comment:
I recognize the work involved in getting here. But. This seems like it should be fine as individual lines, rather than a multiline string... The problem with this approach is that only a failure in the last line will actually cause a failure in the job, making failures somewhat harder to troubleshoot.